src/components/MapItem/index.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  Box,
+  Card,
+  CardContent,
+  CircularProgress,
+  IconButton,
+  Link,
+  Stack,
+  Tooltip,
+  Typography,
+} from '@mui/material'
+import { collection, FieldValue, Timestamp } from 'firebase/firestore'
+import { useAuthState } from 'react-firebase-hooks/auth'
+import { useCollectionData } from 'react-firebase-hooks/firestore'
+import { useIntl } from 'react-intl'
+import { auth, db } from '../..'
+import { msgs } from './messages'
+
+type DateFormatOptionsType = Intl.DateTimeFormatOptions & {
+  dateStyle: 'medium' // This
+}
+
+const SHARE_URL_BASE = 'https://maps-public.mapeo.world/groups'
+
+export const MapItem = ({ id, title, description, createdAt }: MapItemProps) => {
+  const [user] = useAuthState(auth)
+  const observationsRef = user ? collection(db, `groups/${user.uid}/maps/${id}/observations`) : null
+  const [observations = [], observationsLoading] = useCollectionData(observationsRef)
+  const { formatMessage } = useIntl()
+
+  if (!user) return null
+  const dateFormatOptions = { dateStyle: 'medium' } as DateFormatOptionsType
+
+  const shareUrl = `${SHARE_URL_BASE}/${user?.uid}/maps/${id}`
```

Comment:
Is this working for you? I only have one set of data, and it isn't working for that